### PR TITLE
Prepare for release v0.5.3

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Update manifest versions

#### Which issue(s) this PR fixes:

Part of #1711

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by alculquicondor** [comment]
/hold
for #1713

**Comment by alculquicondor** [comment]
Updated release notes based on #1682

File: CHANGELOG/CHANGELOG-0.5.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Fix Kueue crashing at the log level 6 when re-admitting workloads (#1645, @mimowo)
+- Kueue replicas are advertised as Ready only once the webhooks are functional.
+  
+  This allows users to wait with the first requests until the Kueue deployment is available, so that the early requests don't fail. (#1682 #1713, @mimowo @trasc)
+- Remove deleted pending workloads from the cache (#1687, @astefanutti)
```
Comment (mimowo) [inline CHANGELOG/CHANGELOG-0.5.md:13]: I think we should include https://github.com/kubernetes-sigs/kueue/pull/1689. It was cherry-picked and seems important bugfix which should have release note

Comment (mimowo) [inline CHANGELOG/CHANGELOG-0.5.md:13]: Also include https://github.com/kubernetes-sigs/kueue/pull/1617. This was also cherry-picked

**Comment by alculquicondor** [comment]
Added missing ones

**Review by mimowo — COMMENTED** [review]
/lgtm

**Pull request merged**